modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -725,6 +770,10 @@ class CV_EXPORTS_W SimpleBlobDetector : public Feature2D
 
   CV_WRAP static Ptr<SimpleBlobDetector>
     create(const SimpleBlobDetector::Params &parameters = SimpleBlobDetector::Params());
+
+  CV_WRAP virtual void setParams( SimpleBlobDetector::Params params ) { CV_UNUSED(params); return; }
+  CV_WRAP virtual SimpleBlobDetector::Params getParams() { return SimpleBlobDetector::Params(); }
```

Comment:
The methods look confusing. Please remove them if they cannot implement parameters management.